Section 5 -- Validation.md

```diff
@@ -314,18 +310,17 @@ fragment argOnRequiredArg on Dog {
 }
 
 fragment argOnOptional on Dog {
-  isHousetrained(atOtherHomes: true)
+  isHousetrained(atOtherHomes: true) @include(if: true)
 }
 ```
 
-and the following is invalid since command is not defined on DogCommand:
+and the following is invalid since `command` is not defined on `DogCommand`,
```

Comment:
I would split this out into a separate example, since it shows that two things are invalid:

```
fragment invalidFragmentArgName {
  doesKnowCommand(dogCommand: SIT) @include(unless: false)
}
```

and

```
fragment invalidFieldArgName {
  doesKnowCommand(command: CLEAN_UP_HOUSE)
}
```